pkg/apis/provisioning/v1alpha5/requirements.go

```diff
@@ -202,27 +202,30 @@ func (r Requirements) CapacityTypes() stringsets.String {
 }
 
 // Validate validates the feasibility of the requirements.
-func (r Requirements) Validate() (errs *apis.FieldError) {
-	for i, requirement := range r.Requirements {
+//gocyclo:ignore
+func (r Requirements) Validate() (errs []string) {
+	for _, requirement := range r.Requirements {
 		for _, err := range validation.IsQualifiedName(requirement.Key) {
-			errs = errs.Also(apis.ErrInvalidValue(fmt.Sprintf("key %s, %s", requirement.Key, err), "key"))
+			errs = append(errs, fmt.Sprintf("key %s is not a qualified name, %s", requirement.Key, err))
 		}
 		for _, value := range requirement.Values {
 			for _, err := range validation.IsValidLabelValue(value) {
-				errs = errs.Also(apis.ErrInvalidArrayValue(fmt.Sprintf("key %s, value %s, %s", requirement.Key, value, err), "values", i))
+				errs = append(errs, fmt.Sprintf("key %s has an invalid value %s, %s", requirement.Key, value, err))
 			}
 		}
 		if !SupportedNodeSelectorOps.Has(string(requirement.Operator)) {
-			errs = errs.Also(apis.ErrInvalidValue(fmt.Sprintf("%s not in %s for %s", requirement.Operator, SupportedNodeSelectorOps.UnsortedList(), requirement.Key), "operator"))
+			errs = append(errs, fmt.Sprintf("key %s has an unsupported operator %s, supported operators are %s", requirement.Key, requirement.Operator, SupportedNodeSelectorOps.UnsortedList()))
```

Comment:
Why are we moving away from %s not in %s.

Reply:
I was trying `"%s not in %s for key %s ` and it reads `Gt not in [In, NotIn Exists, DoesNotExist] for key XXX`. I feel that the customers may not immediately realize that we are talking about operator violation.  I think it is more readable to say "Key $s has an unsupported operator". 
I am open to change it back.

Reply:
What about `operator %s not in %s for key ...`

Reply:
done

Reply:
done